securedrop/sass/_base.sass

```diff
@@ -20,10 +20,14 @@
 
   h1, h2, h3
     font-weight: bold
+
+  h2, h3
     text-align: center
 
   h1
+    text-align: left
     font-size: 30px
+    color: #004080
```

Comment:
Seems like from the mockups that the text that we want this color and left alignment applied to is actually the h2 text?

![screen shot 2017-01-31 at 2 11 52 pm](https://cloud.githubusercontent.com/assets/7832803/22487164/2551d8a6-e7c1-11e6-97be-5c131fc81ef2.png)